lib/ansible/galaxy/collection.py

```diff
@@ -431,8 +495,15 @@ def from_path(b_path, force, parent=None):
 
         files = info.get('files_file', {}).get('files', {})
 
+        if fallback_metadata and not CollectionRequirement.artifact_info(b_path):
+            skip = False
+        elif fallback_metadata:
+            skip = not force
```

Comment:
Why is this when `not force`? It seems like it is saying skip a SCM collection that contains a `MANIFEST.json` only when `--force` is not set. It seems like we want to still install those collections and `--force` is only for reinstalling collections that are currently installed which is logic outside this function.

Reply:
I'm not sure what I was thinking here. This was to require --force to reinstall a collection from SCM when there was an installed artifact already. The if/elif should probably just change to:
```
if fallback_metadata:
    skip = False
else:
    skip = True
```